main.css

```diff
@@ -0,0 +1,26 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+
+.calculator {
+    width: 320px;
+    height: 520px;
+    margin: 0 auto;
+    top: 20px;
+    position: relative;
+    border: 2px solid;
+    border-color: lightslategray;
+  
+    display: grid;
+    grid-template-columns: 40px 40px 40px 40px;
```

Comment:
If these are all equal and they make up the entire width of the container, you can use `1fr` for each of them.